tests/acceptance/features/webUIUpload/uploadEdgecases.feature

```diff
@@ -99,3 +99,19 @@ Feature: File Upload
     And the user uploads a created file "0" using the webUI
     Then file "0" should be listed on the webUI
     And as "user1" the content of "simple-folder/0" should be the same as the local "0"
+
+  @issue-3015
+  Scenario: Upload a file with a name with same folder name exist
+    Given the user has created folder "new-lorem.txt"
+    And the user has reloaded the current page of the webUI
```

Comment:
Is reloading necessary?

Reply:
yes, we get this error if the UI is not reloaded.
![Screenshot from 2020-02-18 10-10-57 (copy)](https://user-images.githubusercontent.com/27364813/74717617-cf7e9300-5258-11ea-843b-ccfc40be1cc3.png)

Reply:
please make an other issue and PR for that problem. This seems to happen when some folder is created and the UI is not notified about it